pkg/api/types.go

```diff
@@ -30,6 +30,9 @@ type ClusterParams struct {
 	CVOSetupImage              string      `json:"cvoSetupImage"`
 	InternalAPIPort            uint        `json:"internalAPIPort"`
 	RouterServiceType          string      `json:"routerServiceType"`
+        platformProviderType       string      `json:"platformProviderType"`
```

Comment:
The go field should be public (start with a capital)

Reply:
they should also be named with a prefix like `IBMCloud` to indicate they're IBM public cloud specific